test/stdlib/FloatingPointRandom.swift.gyb

```diff
@@ -0,0 +1,139 @@
+// RUN: %target-run-simple-swiftgyb
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+var FloatingPointRandom = TestSuite("FloatingPointRandom")
+defer { runAllTests() }
+
+extension BinaryFloatingPoint where RawSignificand: FixedWidthInteger {
+  // Basic sanity check
+  static func isRandomInRange(_ range: Range<Self>) -> Bool {
+    let x = Self.random(in: range)
+    return range.contains(x)
+  }
+  
+  // Basic sanity check
```

Comment:
Please don't use the term "sanity check", prefer "invariant check" or similar.

Reply:
Done